Dockerfile

```diff
@@ -1,36 +1,32 @@
-FROM openjdk:8u121-jdk-alpine
+FROM openjdk:11-jdk as builder
 
-ARG MAVEN_VERSION=3.5.4
-ARG SHA=ce50b1c91364cb77efe3776f756a6d92b76d9038b0a0782f7d53acf1e997a14d
-ARG BASE_URL=https://apache.osuosl.org/maven/maven-3/${MAVEN_VERSION}/binaries
+# Build song-server jar
+COPY . /srv
+WORKDIR /srv
+RUN ./mvnw clean package -DskipTests
 
-RUN apk add --no-cache curl tar bash \
-  && mkdir -p /usr/share/maven /usr/share/maven/ref \
-  && curl -fsSL -o /tmp/apache-maven.tar.gz ${BASE_URL}/apache-maven-${MAVEN_VERSION}-bin.tar.gz \
-  && echo "${SHA}  /tmp/apache-maven.tar.gz" | sha256sum -c - \
-  && tar -xzf /tmp/apache-maven.tar.gz -C /usr/share/maven --strip-components=1 \
-  && rm -f /tmp/apache-maven.tar.gz \
-  && ln -s /usr/share/maven/bin/mvn /usr/bin/mvn
 
-WORKDIR /usr/src/app
+FROM openjdk:11-jre as server
```

Comment:
replace this with:
`FROM adoptopenjdk/openjdk11:jre-11.0.6_10-alpine as server`